shogun-lib/src/main/resources/graphql/shogun.graphqls

```diff
@@ -12,6 +12,8 @@ type Query {
 
 type Application {
     id: ID
+    created: String
```

Comment:
`Date` is more appropriate here, isn't it?

Reply:
I had issues with configuring this as Date. And as a Date ist transformed to a String for the JSON response anyway this should be ok for now.

Reply:
Alright.